### PR TITLE
Fix deploy rollback on error

### PR DESCRIPTION
**Pull request by erocheleau**
Added a new option to control rollbackOnError for package deploys.
This option must be set to true on production organizations, so the default value is true.

### CODE REVIEW

File: src/commands/deployPackageCommand.ts [hunk 1]
```diff
@@ -91,9 +91,17 @@ export class DeployPackageCommand implements ICommand {
         this.diags.clear();
         this.handleDeployResponse(result);
       })
-      .catch((reason: string) => {
-        if (reason) {
-          vscode.window.showErrorMessage(reason, { title: 'Show output', action: 'SHOW_OUTPUT' }).then((m) => {
+      .catch((ex: any) => {
```
Comment (lbenie) [inline src/commands/deployPackageCommand.ts]: why `ex` over `reason`

reason is more verbose

Comment (erocheleau) [inline src/commands/deployPackageCommand.ts]: because it is not always a reason.
It is an exception of type any instead of just a string.

**Comment by erocheleau** [comment]
Last commit : https://github.com/coveo/vsforce/pull/106/commits/2bea0ae6834c096bc653a20bd11cc920403e08c2

Removed the binary file.

**Comment by lbenie** [comment]
:shipit:

**Pull request merged**